source/ui_strings.c

```diff
@@ -263,6 +263,9 @@ const Language_s language_english = {
         .parental_fail = "Parental Control validation failed!\nBrowser Access restricted.",
         .zip_not_found = "ZIP not found at this URL\nIf you believe this is an error, please\ncontact the site administrator",
         .generic_httpc_error = "Error in HTTPC sysmodule - 0x%08lx.\nIf you are seeing this, please contact an\nAnemone developer on the Theme Plaza Discord.",
+        .http_timeout = "HTTP Timeout.\nCheck your Wi-Fi signal and check\nyour proxy setting if set and try again.",
+        .http_no_network = "Unable to connect to the Internet.\nCheck if the Wi-Fi is enabled and\ncheck your custom DNS if set and try again.",
+        .http_ssl_error = "SSL Error - 0x%08lx.\nCheck if the date/time is correct, and that\nAnemone is up to date. If you're not using\nThemePlaza, the website might not work.",
```

Comment:
Remove reference to Theme Plaza (which should always use a space, for the future).

@astronautlevel2 if you're up for some bikeshedding, I would say this should read
```
"SSL/TLS Error - 0x%08lx.\nCheck if the date/time is correct, and that\nAnemone is up to date."
```
as a failure to negotiate TLS is probably a configuration issue on one side or the other, and Theme Plaza can be misconfigured too.